lib/ansible/cli/galaxy.py

```diff
@@ -1364,7 +1364,8 @@ def _execute_install_collection(
         upgrade = context.CLIARGS.get('upgrade', False)
 
         collections_path = C.COLLECTIONS_PATHS
-        if len([p for p in collections_path if p.startswith(path)]) == 0:
+        collections_path_warnings = C.GALAXY_COLLECTIONS_PATH_WARNINGS
+        if collections_path_warnings and len([p for p in collections_path if p.startswith(path)]) == 0:
```

Comment:
just use the constant directly,  no need to reassign

Reply:
I was trying to be consistent with `collections_path`, but I've done as you asked in the latest push.